sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1313,131 +1313,168 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Computes the cosine inverse of the given value; the returned angle is in the range
-   * 0.0 through pi.
+   * @return angle in radians whose cosine is `e`, as if computed by [[java.lang.Math#acos]]
    *
    * @group math_funcs
    * @since 1.4.0
    */
   def acos(e: Column): Column = withExpr { Acos(e.expr) }
 
+  // scalastyle:off line.size.limit
   /**
-   * Computes the cosine inverse of the given column; the returned angle is in the range
-   * 0.0 through pi.
+   * @return angle in radians whose cosine is `columnName` as if computed by [[java.lang.Math#acos]]
```

Comment:
It's not a big deal, but this change also makes the way this inverse function is described inconsistent with how it's described above. I think it's best to make them the same.